### ISSUE TITLE
Баг в тестах к первому уроку

### ISSUE THREAD

**Issue by DaniilStepanov** [Kotlin-Polytech/Algorithms#14]
Задача "Сортировка температур".  При вызове testGeneratedTemperatures(5000) генерится ~20 миллионов элементов. Домашний компьютер на вызове File("temp_sorted_expected.txt").readLines().joinToString(separator = "\n") с 20-ю миллионами строк падает с out of memory. А котоед падает с java.util.concurrent.TimeoutException, причём подавляющее время работы программы уходит на чтение данных и запись и сравнение результатов

**Comment by mglukhikh** [comment]
Да, скорее всего я погорячился с размерностью. Хотя у меня на локальном компьютере вроде работало. Проверю.

**Comment by mglukhikh** [comment]
Пока замечу, что у меня на домашнем компьютере тесты вполне себе проходят (в том числе для вашего проекта, и даже для проекта с не слишком удачным решением). Видимо, меня сгубила его слишком большая мощность.

**Comment by mglukhikh** [comment]
Как ни странно, если не сливать строки в единый список, а сравнивать их по одной, сильно быстрее это прохождение теста не делает. Константу 5000, конечно, можно уменьшить (например, до 500) и это поможет делу.

**Comment by mglukhikh** [comment]
Выложил пока в репу 2018 снижение константы до 500. При случае поизучаю ситуацию внимательнее

**Comment by mglukhikh** [comment]
Кстати, догадываюсь в чём может быть дело -- на обоих моих компьютерах в связи с Котлином у Идеи имеется много хипа (4 Гб). На вашем, да и на Котоеде тоже, вряд ли есть столько

**Comment by AbdullinAM** [comment]
Уменьшил пока константу до 1000, вроде и по времени и по памяти проходит.
